Takeover request for dynamic-inventory-tags

tenaibms opened the pull request:
https://github.com/BraveH/gear-switch-alert/issues/29 
I asked the author to add one of the main contributors as it doesn't seem like he is very available nowadays, but unfortunately neither responded within a week. Seeing as how the last commit was over a month ago, this plugin fits the criteria for a takeover request, so I am requesting to takeover management of this repository. Please let me know if there is anything I have missed.

YvesW commented:
All recent PRs have been responded to in a day so maybe saying they're both inactive is a bit too much...
@BraveH @daltonpearson please respond to this thread to prevent a plugin takeover.

If you just mean that @daltonpearson should be added to the plugin file here so they can submit PRs to the hub, please let us know because that's something else.

daltonpearson commented:
@YvesW I'm taking a break from OSRS and have shifted to using another gear highlight plugin personally so I would love someone else to take over this plugin. @tenaibms has made at least 1 PR in the past so I give him my blessing. I'm not the original creator so I'd like to hold this for at least a week to give @BraveH a chance if he want's to block this. Keep in mind the plugin currently is 90% functional just with some edge case bugs (which I'll admit are pretty glaring if you use powered staves often).

LlemonDuck commented:
Last call since it's now that it's been a week, I'll merge the takeover tomorrow unless there are any objections.

Felanbird commented:
This fork needs issues to be manually enabled, so users can report problems in the future. 
![380997175-658b6239-0107-425c-8ff2-44bec8c19e77](https://github.com/user-attachments/assets/abedc92f-5289-47cb-8f42-b9c418ac1856)

tenaibms commented:
> This fork needs issues to be manually enabled, so users can report problems in the future.

Done, thank you for letting me know.

And thank you to Dalton Pearson and BraveH for their support. I understand that other things exist in your lives than runelite plugins, so I appreciate you allowing me to maintain it.

YvesW commented:
@BraveH you can also give OP privs on your repo, and add him to the plugin file. That way the repo doesn't need to be bounced around, but OP can just maintain it via your repo.

BraveH commented:
That too is a good idea. I added @tenaibms as contributor to the original repo. You can update there or continue with the takeover, all good by me

LlemonDuck commented:
just let us know which @tenaibms

tenaibms commented:
Since it preserves the existing issues, I think just being added as a contributor would be better. I just wanted to request takeover in case that wasn't possible. I have added the authors tag and changed the link back on the plugin file so that BraveH, myself, and Dalton Pearson can make PRs to update the plugin when necessary, if that works for them.

tenaibms commented:
Good catch, thanks!
